src/reports/components/instance-details.scss

```diff
@@ -51,80 +51,66 @@
     @include fill-available-width;
 
     border-radius: inherit;
-
-    th {
-        padding: 12px 20px;
-
-        vertical-align: top;
-    }
-
-    .row {
-        top: calc(50% - 20px / 2);
-        border-bottom: 0.5px solid $neutral-10;
-    }
-
-    .row:last-child {
-        border-bottom: none;
-    }
-}
-
-.report-instance-table .label {
-    font-size: 14px;
-    line-height: 20px;
-    font-family: $semiBoldFontFamily;
-    color: $primary-text;
-    text-align: left;
-    width: 70px;
-}
-
-.report-instance-table .instance-list-row-content {
-    color: $secondary-text;
-    padding: 12px 20px;
-    font-size: 14px;
-    line-height: 20px;
-    align-items: flex-end;
-    display: flex;
-}
-
-.report-instance-table .instance-list-row-content.content-snipppet {
-    font-family: Consolas, Segoe UI, Courier New, monospace;
-    white-space: normal;
-    word-wrap: break-word;
-    word-break: break-word;
-}
-
-table.report-instance-table {
     border-collapse: collapse;
-    overflow-x: auto;
-    word-break: break-word;
-}
 
-@media screen and (max-width: 480px) {
-    .report-instance-table tr {
-        display: block;
-    }
-
-    .report-instance-table td {
-        display: block;
+    overflow-wrap: break-word;
+    word-break: break-word;
 
-        .label {
-            text-align: left;
+    .row {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+
+        // The padding on the parent and margins on the children are coupled; they
+        // are intended to produce the desired spacing regardless of whether the children
+        // are presented side-by-side or wrapped
+        padding-top: 2px;
+        padding-bottom: 14px;
+        padding-left: 0px;
+        padding-right: 20px;
+        > * {
+            margin-top: 12px;
+            margin-bottom: 0px;
+            margin-left: 20px;
+            margin-right: 0px;
+            padding: 0;
         }
 
-        .instance-list-row-content {
-            text-align: right;
+        border-bottom: 0.5px solid $neutral-10;
```

Comment:
nit: we can also write this like so:

```
        + .row {
            border-top: 0.5px solid $neutral-10;
        }
```

Adjacent elements can have styles applied to them like ($selector + $selector) to apply styles to all but the first element (hence why border-top over border-bottom). Yes, the nittiest of nits but I always like sharing this cool selector usage. Link to some text on it: https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/

Reply:
I agree that this would work equally well, but I didn't use it because I think it's significantly harder for a reader to reason about, despite being shorter:

* if someone don't know the advanced selectors off the top of their head, "last-child" is a lot easier to infer the meaning of than "+"
* even if I do remember that "+" is an adjacent sibling selector, I have to remember whether it refers to the "left" or "right" sibling to understand if the css should be using -top or -bottom in a code review